src/app/store/reducers/app-config.reducer.ts

```diff
@@ -9,17 +9,27 @@ export interface AppConfigState {
   termsAndCondition: TermsAndCondition;
   loaded: boolean;
   loading: boolean;
+  locationInfo: LocationInfo;
   termsAndConditions: TCDocument;
   isTermsAndConditionsFeatureEnabled: boolean;
   useIdleSessionTimeout: boolean;
   userDetails: UserDetails;
 }
 
+export interface LocationInfo {
+  primaryLocation: string;
+  caseId: string;
+  jurisdiction: string;
+  region: string;
+  isCaseAllocator: boolean;
+}
+
 export const initialState: AppConfigState = {
   config: {},
   termsAndCondition: { isLoaded: false, hasUserAcceptedTC: false },
   loaded: false,
   loading: false,
+  locationInfo: null,
```

Comment:
locationInfo default value could blank array [] instead of null

Reply:
Done this